x-pack/plugins/ml/public/application/timeseriesexplorer/timeseriesexplorer.js

```diff
@@ -1186,6 +1253,42 @@ export class TimeSeriesExplorer extends React.Component {
                 />
               );
             })}
+            {actualPlotFunction && (
+              <EuiFlexItem style={{ textAlign: 'right' }}>
+                <EuiFormRow
+                  label={i18n.translate('xpack.ml.timeSeriesExplorer.metricPlotByOption', {
+                    defaultMessage: 'Function',
+                  })}
+                >
+                  <EuiSelect
```

Comment:
Is it possible to decrease the width on this element?

![image](https://user-images.githubusercontent.com/7405507/97477681-d3b29280-1947-11eb-90b1-b52749d324a2.png)

Reply:
Decreased the width here  https://github.com/elastic/kibana/pull/81662/commits/21bb6cb31d31959d096b318b15f2426c840d6ac9

<img width="734" alt="Screen Shot 2020-11-03 at 14 09 12" src="https://user-images.githubusercontent.com/43350163/98035510-3d1a2180-1dde-11eb-8f6f-5b15246a041c.png">